thunder/tests/opinfos.py

```diff
@@ -3595,6 +3595,51 @@ def make_idx(dim_length: int, indices: int):
     idx0 = make_idx(7, 9)
     yield SampleInput(a, (Ellipsis, idx0))
 
+    # list indexing
```

Comment:
let's also add a couple cases to the basic indexing cases where:

- all slices are from 0 to the length of their dimension (and so trigger the performance optimization) and have stride of 1
- all slices are from 0 to the length of their dimension (and so trigger the performance optimization) and have strides != 1

Reply:
Let me add this :) thank you!